src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -28,10 +28,20 @@ const professionalClient = (() => {
         }
 
         const status = State.getResponse('get_account_status.status') || [];
-        if (is_in_page && /professional/.test(status)) {
+        if (is_in_page && status === 'professional') {
```

Comment:
this doesn't work, status is an array.

Reply:
I'll revert it back to regex. I think that works.